governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/SolanaStakingMultisigInstruction.ts

```diff
@@ -54,6 +55,31 @@ export class SolanaStakingMultisigInstruction implements MultisigInstruction {
         case "Authorize":
         case "AuthorizeWithSeed":
         case "Delegate":
+          const decodedDelegate = StakeInstruction.decodeDelegate(instruction);
+          return new SolanaStakingMultisigInstruction(
+            "Delegate",
+            {},
```

Comment:
Now Authorize and AuthorizeWithSeed with also run the "Delegate" code since there are no breaks. Also "Deactivate" will also run the Delegate code. I think this was unintentional. We should move the Delegate above the Authorize and add break for both "Deactivate" and "Deligate"

Reply:
Great catch. Not adding `break` because it returns inside the case block